config/params/mainnet_config.go

```diff
@@ -188,6 +188,9 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	BeaconStateFieldCount:          21,
 	BeaconStateAltairFieldCount:    24,
 	BeaconStateBellatrixFieldCount: 25,
+	OrphanLateBlockFirstThreshold:  4,
```

Comment:
We're already adding a few new globals, might as well add the required fraction of committee votes for orphaning late blocks here

Reply:
I was worried about this, doing so requires calling a function from the forkchoice package to get back the global parameters and then request the constant. This constant can only be useful from within forkchoice. Is it really worth it to have it as a beacon-chain parameter?

Reply:
Fair enough, perhaps we can just give it a more descriptive name to know that it is the safe number of committee votes to enable proposer reorgs